ideascube/mediacenter/management/commands/clean.py

```diff
@@ -37,14 +38,28 @@ def add_arguments(self, parser):
             '--type', action='append', choices=document_types,
             help='The type of media to remove, e.g "--types=image". Can be '
                  'specified multiple times to clean multiple types.')
+        clean_media.add_argument(
+            '--created-before', type=date_argument,
+            help='Only remove media created before this date.')
+        clean_media.add_argument(
+            '--created-after', type=date_argument,
+            help='Only remove media created after this date.')
```

Comment:
Maybe inverse the two options order. It seem more natural when specifying a range to give first the start (`--created-after`) of the range and then the end (`--created-before`). And this is the order you've used when you're filtering the queryset.

You could also improve the help the `create-after` to explicit that it is media created after **or at** this date. (`created_at__gte`)